internal/host/refresh_status_preprocessor.go

```diff
@@ -130,6 +156,15 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *validationConte
 		}
 	}
 
+	// Now that all validations have been executed, skip the validations that we wish to ignore.
+	for _, currentResult := range validationsOutput {
```

Comment:
same question as will cluster part

Reply:
This block is here for a reason and in attempting to resolve this comment I am quite sure that it should be here.

The issue is that we are not applying the ignore at the very last moment that anything could alter a condition and in testing, I have just demonstrated that this concern is justified by implementing your suggestion.

So, in the lack of any further suggestions, I opt to leave this code block here. Open to reviewing the block itself to simplify, but not to removing it.